.vscode/settings.json

```diff
@@ -11,6 +11,6 @@
     "dist/": true
   },
   "editor.tabSize": 2,
-  "editor.formatOnSave": true,
+  "editor.formatOnSave": false,
```

Comment:
For those who like this, feel free to update VSCode editor settings via `cmd + ,` -> 'formatOnSave'.